### PR TITLE
Import support

### PR DESCRIPTION
**Pull request by uittorio**
I've create a couple of test for import alias. @Pmyl you've mentioned that something was not supported. Could you write down the example so I will update the test and fix the issue in this PR? :)

### CODE REVIEW

**Comment by Pmyl** [comment]
ImportEqualDeclaration is not supported yet

You will have it for example when using an interface in a namespace without using the entire namespace, so instead of doing:

createMock<Namespace.Interface>();

we can do

import Interface = Namespace.Interface;
createMock<Interface>();



I think we never covered import star and require

**Comment by uittorio** [comment]
Okay, ill create few test :) thanks

File: test/transformer/descriptor/utils/namespace/namespace.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
+var NameSpaceInterfaceImport;
+(function (NameSpaceInterfaceImport) {
+    var Enum;
```
Comment (Pmyl) [inline test/transformer/descriptor/utils/namespace/namespace.js]: Why is this file pushed?

Comment (uittorio) [reply test/transformer/descriptor/utils/namespace/namespace.js]: Ahah by mistake, now im using git add by commandline and sometimes I forget to exclude the one I dont want! Ill remove it :)

File: test/transformer/descriptor/create-mock-list-values.test.ts [hunk 2]
```diff
@@ -1,41 +0,0 @@
-import { createMockList } from 'ts-auto-mock';
```
Comment (Pmyl) [inline test/transformer/descriptor/create-mock-list-values.test.ts]: Didn’t you remove these tests in a different commit?

File: test/transformer/descriptor/utils/namespace/namespace.js (hunk 1 above)
Comment (uittorio) [reply test/transformer/descriptor/utils/namespace/namespace.js]: Removed

File: test/transformer/descriptor/create-mock-list-values.test.ts (hunk 2 above)
Comment (uittorio) [reply test/transformer/descriptor/create-mock-list-values.test.ts]: I don't think see. the duplicated test is inside test/transformer/create-mock-list-values.test.ts and create-mock-list.test.ts

**Pull request merged**